src/utils.ts

```diff
@@ -65,7 +65,17 @@ export function createApolloClient(options: {
   //     wsorhttplink
   //   ])
   const client = new ApolloClient({
-    cache: new InMemoryCache(),
+    cache: new InMemoryCache({
+      cacheRedirects: {
+        Query: {
+          dao: (_, args, { getCacheKey }) =>  getCacheKey({ __typename: 'DAO', id: args.id }),
```

Comment:
why is this needed?

Reply:
This is how apollo cache works.
If you call a list say daos[], and later call a specific dao, it will not get it from the cache, unless you do a cache Redirect. Not sure why, I was surprised by it, but that is what I found in the documentation.